website/src/components/pageStateSelectors/FallbackElement.tsx

```diff
@@ -0,0 +1,38 @@
+export function SequencingEffortsSelectorFallback() {
+    return <div className='h-[208px] lg:h-auto'></div>;
+}
+
+export function AnalyzeSingleVariantSelectorFallback() {
+    return <div className='h-[372px] lg:h-auto'></div>;
+}
+
+export function CompareSideBySideSelectorFallback({ numFilters }: { numFilters: number }) {
+    return (
+        <div>
+            {numFilters > 2 && <div className='h-[264px]' />}
+            {numFilters <= 2 && <div className='h-[220px]' />}
```

Comment:
Why does this actually depend on `numFilters`?

Reply:
The date range selector changes it height: For 2 filters it is in one row and for more filters it switches to the two row design.

Reply:
Or is this only the case for me?

Reply:
I didn't check in detail, but the layout here is difficult anyway. Also the mutation filter will change its height depending on how many mutations are selected. Anything close to the actual value is good enough IMO.